contrib/fsl_integration/sct_plugin.py

```diff
@@ -808,11 +808,35 @@ def run_main():
     panel_vlb = TabPanelVertLB(parent=notebook)
     panel_reg = TabPanelRegisterToTemplate(parent=notebook)
 
-    notebook.AddPage(page=panel_propseg, caption="sct_propseg", select=True)
-    notebook.AddPage(page=panel_sc, caption="sct_deepseg_sc", select=False)
-    notebook.AddPage(page=panel_gm, caption="sct_deepseg_gm", select=False)
-    notebook.AddPage(page=panel_vlb, caption="sct_label_vertebrae", select=False)
-    notebook.AddPage(page=panel_reg, caption="sct_register_to_template", select=False)
+    try:
+        notebook.AddPage(page=panel_propseg, caption="sct_propseg", select=True)
+        notebook.AddPage(page=panel_sc, caption="sct_deepseg_sc", select=False)
+        notebook.AddPage(page=panel_gm, caption="sct_deepseg_gm", select=False)
+        notebook.AddPage(page=panel_vlb, caption="sct_label_vertebrae", select=False)
+        notebook.AddPage(page=panel_reg, caption="sct_register_to_template", select=False)
+    except TypeError as e:
+        # Partially addresses https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3988
+        if "unexpected type 'float'" in str(e):
+            sct_view_panel.removeFromFrame()
+            raise EnvironmentError(
+                "The FSL installation contains an out of date package: 'wxpython' (4.1.1). "
```

Comment:
I'd be wary of guessing that the version is 4.1.1 without checking. But programmatically getting the actual version information is probably tricky. Can we just leave out the version number?
```suggestion
                "The FSL installation contains an out of date package: 'wxpython'. "
```